src/instrumenter/callgraph.ts

```diff
@@ -33,6 +35,25 @@ export interface CallGraph {
     overridenBy: FunMap;
 }
 
+function findCallersAndCallees(fun: FunctionDefinition, callers: FunMap, callees: FunMap): void {
```

Comment:
Can we rename this function to `processCallNodes` ?

Reply:
Sure, will do.